src/harness/reference_models/move_list_calc/move_list.py

```diff
@@ -0,0 +1,435 @@
+                                            # uniform distribution over [0,1)
+    reliabilities = np.append(reliabilities, [0.5])  # add 0.5 (for median loss) as
+                                            # a last value to reliabilities array
+    results = wf_itm.CalcItmPropagationLoss(lat_cbsd, lon_cbsd, h_cbsd, lat_c, lon_c, h_c,
+                                            reliability=reliabilities,
+                                            freq_mhz=center_freq_mhz)
+    path_loss = np.array(results.db_loss)
+    # If the CBSD is indoors, account for attenuation due to building loss [R2-SGN-03]
+    if indoor_cbsd:
+        path_loss += BLDG_CLUTTER_LOSS
+
+    # Calculate the interference contributions
+    interf = eirp_cbsd - path_loss
+    median_interf = interf[-1]      # last element is the median interference
+    K_interf = interf[:-1]          # first 'K' interference
+
+    # Compute bearing from protection constraint c to the CBSD for later usage
+    _, bearing_c_cbsd, _ = vincenty.GeodesicDistanceBearing(lat_c, lon_c, lat_cbsd,
+                                                            lon_cbsd)
+
+    # Store interference contributions		
+    interference = {
+        'grantIndex': grant_index,
+        'medianInterference': median_interf,
+        'randomInterference': K_interf,
+        'bearing_c_cbsd': bearing_c_cbsd}
+
+    return interference
+
+
+def formInterferenceMatrix(cbsd_grant_list, constraint, num_iter):
+
+    """
+    Form the matrix of interference contributions to protection constraint c.
+
+    Inputs:
+        cbsd_grant_list:    a list contains cbsd grants
+        constraint:         a dictionary contains information of protection constraint (c)
+        num_iter:           number of random iterations
+
+    Returns:
+        I:                  a matrix of interference contributions to protection
+                            constraint (c) with dimensions (num_iter x len(cbsd_grant_list))
+        sorted_grant_index: a list of indices of the sorted grant list
+        sorted_bearing:     a list of bearings from protection point to CBSDs of the
+                            sorted grant list
+    """
+
+    # Compute interference contributions of each grant to the protection constraint
+    interf_list = []
+    for cbsd_grant in cbsd_grant_list:
+        interf = computeInterference(cbsd_grant, constraint, num_iter)
+        interf_list.append(interf)
+
+    # Sort grants by their median interference contribution, smallest to largest
+    interf_list.sort(key=itemgetter('medianInterference'))
+
+    # Get indices, bearings, interference contributions of the sorted grants
+    sorted_grant_index = [i.get('grantIndex') for i in interf_list]
+    sorted_bearing = [i.get('bearing_c_cbsd') for i in interf_list]
+    I = np.array([i.get('randomInterference') for i in interf_list]).transpose()
+
+    return I, sorted_grant_index, sorted_bearing
+
+
+def grantsInMainBeam(ant_az, bearing):
```

Comment:
Function useless if using the `antenna.GetRadarNormalizedAntenna()` routine.
See a comment below.